execution.bs

```diff
@@ -3161,8 +3181,11 @@ class in_place_stop_token;
 class in_place_stop_source;
 
 // [stopcallback.inplace], class template in_place_stop_callback
-template&lt;typename Callback>
-class in_place_stop_callback;
+template&lt;class CB>
+  class in_place_stop_callback;
+
+template&lt;class T, class CB>
+  using stop_callback_for_t = typename T::template callback_type&lt;CB>;
```

Comment:
LWG didn't ask for this change, but this type alias tidies up the spec in several places and seems generally useful, so I made it normative. I can back out if requested.